src/lib/pickles/common.ml

```diff
@@ -235,23 +235,64 @@ let tick_public_input_of_statement ~max_proofs_verified ~feature_flags
     (Backend.Tick.Field.Vector.length input)
     ~f:(Backend.Tick.Field.Vector.get input)
 
+let maybe_scale ~scale g x l =
```

Comment:
Any function with `maybe_` definitely should have a comment explaining... ;-)

Reply:
This is actually unused.

Reply:
Interesting, why doesn't it complain about no `_` prefix?

Reply:
Because #12120 has not made it yet :-) Thus no warnings in Pickles 🥳